internal/node/services.go

```diff
@@ -20,26 +20,14 @@ type portOffset = int
 const (
 	portOffsetProxy = iota
 	portOffsetAuthorityClaimer
-	portOffsetRedis
 	portOffsetPostgraphile
 )
 
-const localhost = "127.0.0.1"
-
 // Get the port of the given service.
 func getPort(c config.NodeConfig, offset portOffset) int {
 	return c.HttpPort + int(offset)
 }
 
-// Get the redis endpoint based on whether the experimental sunodo validator mode is enabled.
-func getRedisEndpoint(c config.NodeConfig) string {
-	if c.ExperimentalSunodoValidatorEnabled {
```

Comment:
Removing Redis also removes this, since it's impossible for us to support this feature without it. We can move removing Redis to different PR to work things out.

Reply:
Don't worry, that was a reminder for cleaning up other parts of the code that still refer to the experimental configuration